wolfssl/wolfcrypt/visibility.h

```diff
@@ -50,7 +50,13 @@
         #define WOLFSSL_API
         #define WOLFSSL_LOCAL
     #endif /* HAVE_VISIBILITY */
-#else /* BUILDING_WOLFSSL */
+
+    #ifdef WOLFSSL_LOCALIZE_TEST_APIS
```

Comment:
Misleading name - says to me we have test APIs we are making local not APIs making local as we are not testing.

Reply:
another suggestion on naming:
instead of `WOLFSSL_TEST_API` can we use `WOLFSSL_TEST_VISIBLE` ? imo it makes clear that the function is *not* a public API

Reply:
done.  went with `WOLFSSL_TEST_VIS`.